__tests__/accessiblity-audit.test.js

```diff
@@ -16,6 +17,8 @@ async function analyze (page, path) {
     .exclude('.govuk-skip-link')
     // axe reports that the back to top button is not inside a landmark, which is intentional.
     .exclude('.app-back-to-top')
+    // axe reports that the Browsersync banner is not inside a landmark, which is intentional.
+    .exclude('#__bs_notify__')
```

Comment:
If you've already run `npm start` then http://localhost:3000 will have the Browsersync banner during tests

This line makes sure we exclude it from the accessibility checks